.gitignore

```diff
@@ -0,0 +1,8 @@
+mvc/vendor/
+*.jpg
+*.png
+*.ico
```

Comment:
If the images are in a single folder, add the folder path in .gitignore files instead of mentioning individual extensions.

Reply:
Although this is fine for the task, please keep in mind that you might need images for some styling purposes but this will ignore all files with the given extensions across all folders. So we should add these files in gitignore.

Reply:
Okay , I use  `public/assets/img/` folder for storing the user uploaded image, then should I write in `gitignore` like this `public/assets/img/*jpg` to ignore those images , and store the images for styling in another folder ?